README.md

```diff
@@ -1,2 +1,202 @@
-# saritasa-github-actions
-GitHub actions
+## CI-wordpress reusable workflow
```

Comment:
this README should be documentation for all workflows. 

As such you structure should be more elegant

```md
# Workflows
## android
## asp.net
## ios
## wordpress
```

and here have you docs

Reply:
done